src/components/product-progress/step-progress-extractor.service.ts

```diff
@@ -60,12 +62,19 @@ const parseProgressRow =
       const percentDecimal = cell.asNumber;
       return percentDecimal ? percentDecimal * 100 : undefined;
     };
-    const steps = entries(stepColumns).map(
-      ([step, column]): StepProgressInput => ({
-        step,
-        completed: progress(column),
-      })
+    const steps = entries(stepColumns).flatMap<StepProgressInput>(
+      ([step, column]) => {
+        // To get the fiscal year is needed to check the next column
+        const fiscalYear = sheet.cell(column.next(), row).asNumber;
+        const fullFY = fiscalYear ? fullFiscalYear(fiscalYear) : undefined;
+        // Only include step if it references a fiscal year within the project
+        if (!fullFY || !projectFiscalYears.intersection(fullFY)) {
```

Comment:
This isn't exactly right either. If there's no fiscal year the product could still be in progress like `50%` or it could be blank meaning not started yet. I think really the only way to know if the step is planned is to look at the planning sheet. We can reuse the same logic from the product extractor to determine whether the step is planned or not for each row.